events/entry.go

```diff
@@ -40,11 +48,59 @@ var getDockerClient = func() (*docker.Client, error) {
 	return NewDockerClient(false)
 }
 
-var getHandlers = func(dockerClient *docker.Client) map[string]Handler {
-	handler := &StartHandler{
+var getHandlers = func(dockerClient *docker.Client) (map[string]Handler, error) {
```

Comment:
Poor man's dependency inject. I can override these in tests:
https://github.com/rancherio/host-api/pull/22/files#diff-6f392d76280b6c1b00c76d7866004664R33